src/common/config.h.cmake

```diff
@@ -12,8 +12,6 @@
 #define HAVE_CFLOCALECOPYCURRENT 1
 #endif
 
-#cmakedefine GLEW_STATIC
```

Comment:
I think the GLEW headers did actually require this to work. Does the new code set it somewhere else?

Reply:
Yes. I was getting a lot of warnings with the new FindGLEW that GLEW_STATIC is defined twice. Don't ask me how and where it is defined then though. I suspect there's some logic about it in the FindGLEW module. The static targets have probably set INTERFACE_COMPILE_DEFINITIONS correctly so we don't have to add definition by hand.

Also, I recall I gave the explanation for the change in the related commit message.

Reply:
Oh yeah, you did, I just didn't really look at the commit messages as I assumed all important things are in the PR summary.

Reply:
I looked into this. I don't see FindGLEW defining GLEW_STATIC anywhere. I suspect vcpkg is actually setting it and it's not related to the new FindGLEW module at all. This remind me that this PR should be tested with MSYS2. Might be one of the reasons it didn't build with MXE actually...

What should've be done here is wrapping this definition around `#ifdef GLEW_STATIC` to avoid the "macro redefinition" warnings.

Reply:
Adding MSYS2 builds to CI too may be a good idea. I think it even comes preinstalled on GH Actions Windows runners.

If this is the only problem with MXE builds, we can try ti bring it back until the OpenAL issue is fixed.

Reply:
I think I'll leave MSYS2 builds for another PR. I'd like to introduce an option to install DLLs to CMake first.